lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -482,7 +503,7 @@ _LoadTexture(const std::string& path, bool& isColorSpaceSRGB, MFloatArray& uvSca
 #if USD_VERSION_NUM >= 2102
     HioImageSharedPtr image = HioImage::OpenForReading(path);
 #else
-    GlfImageSharedPtr     image = GlfImage::OpenForReading(path);
+    GlfImageSharedPtr image = GlfImage::OpenForReading(path);
```

Comment:
@kxl-adsk Any idea why github clang-format-linter doesn't like this change?

Reply:
It is one more mysterious mystery about clang format. I will revert this change anyway so we don't have to wait on it.

Reply:
I've logged the problem in the wiki.